tests/TaskLineRenderer.test.ts

```diff
@@ -662,3 +662,21 @@ ${task.toFileLineString()}
         await renderAndVerifyHTML(minimalTask, layoutOptionsShortMode());
     });
 });
+
+describe('Field Layout Detail tests', () => {
+    it('should supply a class name and a data attribute name', () => {
+        const fieldLayoutDetail = new FieldLayoutDetail('stuff', 'taskAttribute', () => {
+            return '';
+        });
+        expect(fieldLayoutDetail.className).toEqual('stuff');
+        expect(fieldLayoutDetail.attributeName).toEqual('taskAttribute');
+    });
+
+    it('should calculate data attribute value', () => {
+        const fieldLayoutDetail = new FieldLayoutDetail('foo', 'bar', () => {
+            return 'someValue';
+        });
+        const attributeValue = fieldLayoutDetail.attributeValueCalculator('createdDate', new TaskBuilder().build());
+        expect(attributeValue).toEqual('someValue');
+    });
+});
```

Comment:
How much value do you think these tests have now?

Reply:
I think they show well the behaviour of the FLD class and they kept its behaviour unchanged while I was changing the class =)